server/http/controller_test.go

```diff
@@ -120,6 +121,7 @@ func setupController(t *testing.T) controllerFixture {
 			nil,
 			nil,
 			mappings.New(traces.NewConversionConfig(), comparator.DefaultRegistry(), mdb),
+			environment.NewRepository(nil),
```

Comment:
referecing my comment about depending on an interface, this could also be just `nil` instead of an invalid repo, which reads nicer I think (personal opinion)